deploy/clowdapp.yaml

```diff
@@ -22,6 +22,52 @@ objects:
       testing:
         iqePlugin: provisioning
       deployments:
+        - name: background-job-worker
```

Comment:
the name is quite long, do we insist on long name, wouldn't `worker` work as well?
I'm just worried of having `provisioning-backend-background-job-worker-xxxxxxxxxxxxxxxxxxx` as pod name, but if you feel it is superior to worker by a lot, lets do it ^_^

Reply:
Sure I saw similar name in sources, but let's make it shorter good point.